src/test/java/com/accesspoint/rulesengine/apitests/ruleset/RulesetTest.java

```diff
@@ -0,0 +1,77 @@
+package com.accesspoint.rulesengine.apitests.ruleset;
+
+import com.accesspoint.rulesengine.entity.Ruleset;
+import com.accesspoint.rulesengine.repository.RulesetRepository;
+import io.restassured.RestAssured;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.context.junit4.SpringRunner;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.TimeZone;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+
+public class RulesetTest {
+
+    @LocalServerPort
+    private Integer port;
+
+    @MockBean
+    private RulesetRepository rulesetRepository;
+
+    @Before
+    public void init() {
+        RestAssured.baseURI = "http://localhost";
+        RestAssured.port = this.port;
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    }
+
+    @Test
+    public void StatusCode200() {
```

Comment:
Here are some resources for better test names.

- https://www.baeldung.com/java-unit-testing-best-practices
- https://www.softwaretestingmagazine.com/knowledge/how-to-choose-the-right-name-for-unit-tests/
- https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea